.evergreen/config.yml

```diff
@@ -36,60 +36,51 @@ functions:
     # Make an evergreen exapanstion file with dynamic values
     - command: shell.exec
       params:
-        working_dir: "src"
+        shell: bash
         script: |
-           echo "" > expansion.yml
-
+           pushd src
            # Get the current unique version of this checkout
            if [ "${is_patch}" = "true" ]; then
               VERSION=$(git describe)-patch-${version_id}
            else
               VERSION=latest
            fi
-           echo "CURRENT_VERSION: $VERSION" >> expansion.yml
-
-           # Python has path problems on Windows. Detect prospective mongo-orchestration home directory
-           OS=$(uname -s | tr '[:upper:]' '[:lower:]')
-           case "$OS" in
-              cygwin*)
-                 # Python has problems with unix style paths in cygwin. Must use c:\\ paths
-                 export DRIVERS_TOOLS="c:/drivers-tools"
-                 ;;
-              *)
-                 export DRIVERS_TOOLS="$PWD/../drivers-tools"
-                 ;;
-           esac
+
+           export TASK_DIR=`pwd`
+           if [ "Windows_NT" == "$OS" ]; then
```

Comment:
I think it's populated in cygwin bash; this is pulled from the evergreen self-tests, and seems to be the prevailing idiom.